src/pages/ja/amman/configuration.md

```
@@ -0,0 +1,126 @@
+    ...
+   deactivateFeatures: ['21AWDosvp3pBamFW91KB35pNoaoZVTM7ess8nr2nt53B'],
+  }
+}
+```
+
+**注意**: 上記のプロパティのうち1つのみを設定できます
+
+#### リソース
+
+- [テストバリデータランタイム機能](https://docs.solana.com/developing/test-validator#appendix-ii-runtime-features)
+- [ランタイム新機能](https://docs.solana.com/developing/programming-model/runtime#new-features)
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)